packages/io/src/router.ts

```diff
@@ -9,50 +10,63 @@ export function isRoute(x: unknown): x is Route {
 }
 
 function matchPath(path: Path, route: Route): boolean {
-  // todo: add more match rule
-  return path === route.path
+  return route.regexp.test(path)
+}
+
+function makeRoute(options: { path: Path; middleware: Middleware<RouterContext>; method: Method }): Route {
```

Comment:
Function name `makeRoute` is hard to understand.

It's an internal function, so we can name it a long but detailed text